src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala

```diff
@@ -48,9 +48,10 @@ object FireCloudApiService extends LazyLogging {
         if (logger.underlying.isDebugEnabled) {
           logger.debug(e.getMessage, e)
         } else {
-          logger.error(e.getMessage)
+          logger.error(s"$e")
         }
-        complete(StatusCodes.InternalServerError -> ErrorReport(e))
+        // ErrorReport.apply with "message" kwarg. is specifically used to mute Stack Trace output in HTTP Error Responses
+        complete(StatusCodes.InternalServerError -> ErrorReport(message="An error occurred. Please see logs for more details."))
```

Comment:
I think `ErrorReport(e.toString)` or `ErrorReport(e.getMessage)` is a good middle ground here - we'd still give the end user *some* information about what error occurred, but we'd still be leaving out the stack trace.